nextflow_schema.json

```diff
@@ -266,6 +266,40 @@
                     "help_text": "At least the type of reference sequences should be specified. (For RNA sequences, use the DNA setting) - from [MALT manual](https://software-ab.cs.uni-tuebingen.de/download/malt/).\n\nParameters must be wrapped in _both_ single and then double quotes outside these, to ensure the parameters are interpreted as a string for internal use rather than interpreted by the terminal as a pipeline-level parameter.",
                     "default": "--sequenceType DNA",
                     "fa_icon": "fas fa-users-cog"
+                },
+                "build_sourmash_dna": {
+                    "type": "boolean",
+                    "fa_icon": "fas fa-toggle-on",
+                    "description": "Whether to build a sourmash reference from the provided nucleotide sequences.",
+                    "help_text": "Sourmash supports both nucleotide and amino acid sequences. With this flag, you determine that a reference database should be built from all the nucleotide FastA files defined in the sample sheet."
+                },
+                "sourmash_build_dna_options": {
+                    "type": "string",
+                    "default": "--param-string \\'scaled=1000,k=31,noabund\\",
+                    "fa_icon": "fas fa-users-cog",
+                    "description": "Must be a single parameter string passed to sourmash sketch dna.",
```

Comment:
Note to self: rephrase for consistency with other options parameters (do same with AA too)